spvc/test/run_spirv_cross_tests.py

```diff
@@ -45,127 +43,112 @@
 test_count = 0
 pass_count = 0
 
-not_used, tmpfile = tempfile.mkstemp()
-devnull = open(os.devnull, 'w')
-
-def mkdir_p(filepath):
-    ''' make sure directory exists to contain given file '''
-    dirpath = os.path.dirname(filepath)
-    try:
-        os.makedirs(dirpath)
-    except OSError as exc:
-        if not (exc.errno == errno.EEXIST and os.path.isdir(dirpath)):
-            raise
+def check_call(cmd):
```

Comment:
The methods should have method comments.

This became acute for me when reading the spvc function (since this version of the code has mixed responsibility: to run the command *and* increment the test count.)

Reply:
done